lib/thunderbird_common.pm

```diff
@@ -153,10 +153,12 @@ sub tb_send_message {
             assert_and_click "thunderbird_focus_security_exception-TW";
             assert_and_click "thunderbird_select_sentemail_window-TW";
         }
+
         assert_and_click "thunderbird_maximized_send-message";
+        assert_screen "thunderbird_sent-folder-appeared", 180;
     }
     else {
-        assert_screen 'thunderbird_sent-folder-appeared';
+        assert_screen "thunderbird_sent-folder-appeared", 90;
```

Comment:
please keep single ticks by default unless you want to expand perl variables within strings

Reply:
ok, I will keep in mind to use simple ticks. tks